File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/cells/GetMinDataColumnWorksheet.java
package com.aspose.cells.cloud.examples.cells;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class GetMinDataColumnWorksheet {

	public static void main(String... args) throws IOException {
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			String input = "sample1.xlsx";
			Path inputFile = Utils.getPath(GetMinDataColumnWorksheet.class, input);

			String sheetName = "Sheet1";
			String cellOrMethodName = "mindatacolumn";

			storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

			Integer apiResponse = cellsApi.GetWorksheetCellProperty(input, sheetName, cellOrMethodName, Utils.STORAGE,
					Utils.FOLDER);
			System.out.println(" MinDataColumn :: " + apiResponse);
		}

		catch (Exception e) {
			e.printStackTrace();
		}

	}
}

File: Examples/Node.js/SDK/Cells/GetMinDataColumnWorksheet.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var CellsApi = require('asposecellscloud');
var configProps = require('../Config/config.json');
var data_path = '../../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose Cells API SDK
var cellsApi = new CellsApi(config);

// Set input file name
var name = "Sample_Test_Book.xls";
var sheetName =  "Sheet2";
var cellOrMethodName = "mindatacolumn";
try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, null, null, data_path + name , function(responseMessage) {

	assert.equal(responseMessage.status, 'OK');

	// Invoke Aspose.Cells Cloud SDK API to get index of first column which contains data in a worksheet
	cellsApi.GetWorksheetCell(name, sheetName, cellOrMethodName, null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			console.log("MinDataColumn :: " + responseMessage.body);

			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1